project/appweb/src/main/java/web/Start.java

```diff
@@ -0,0 +1,28 @@
+package web;
+
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+import java.io.PrintWriter;
+
+@WebServlet(value = "/about")
+public class Start extends HttpServlet {
+
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp) throws ServletException, IOException {
+
```

Comment:
Имеет смысл определять ContentType и CharacherEncoding
```suggestion
        resp.setContentType("text/html");
        resp.setCharacterEncoding("UTF-8");
```